code/__DEFINES/pai.dm

```diff
@@ -0,0 +1,31 @@
+/// Time between fold out
+#define HOLOCHASSIS_COOLDOWN 5 SECONDS
```

Comment:
```suggestion
#define HOLOCHASSIS_COOLDOWN (5 SECONDS)
```
generally we encase time defines in brackets because it can cause order of operation issues. ad nauseum for the rest of this file